opentelemetry-otlp/CHANGELOG.md

```diff
@@ -2,8 +2,11 @@
 
 ## vNext
 
-- **Breaking** [1994](https://github.com/open-telemetry/opentelemetry-rust/pull/1994) The logrecord event-name is added as attribute with
-key `name` only if the feature flag `populate-logs-event-name` is enabled.
+- **Breaking**
+[1994](https://github.com/open-telemetry/opentelemetry-rust/pull/1994) The
```

Comment:
It would be nice to add the link to this PR as well as this is the PR which uses `event.name` as the attribute name.